content/tutorials/styleguide_r_code/index.Rmd

```diff
@@ -307,6 +307,15 @@ if (requireNamespace("lintr", quietly = TRUE)) {
 }
 ```
 
+#### Data
```

Comment:
I do think all these recommendations handle about data related to package development, so I would explicitly mention this here as this is not applicable for _analysis oriented_ work without any package development?

Reply:
Indeed, these recommendations handle on where to put data in packages, that's why I put them in the section `R package` under `File structure`.

I'm not sure how to change this.  The difference between a 3th and a 4th level title is not very pronounced, which could indeed lead to confusion on the exact position of the section.  But looking at it from a distance: maybe the main problem is that the guideline is growing too long for one page?  ... (I'll continu this discussion in issue #118)

I could change the title `Data` in `How to store data in a package`, but then it contrasts with the short titles of the other subsections of `R package` (which are nice in my opinion).

Reply:
As it is part of the main section about packages, I do agree. Fine for me to keep it as such.